helios-testing/src/main/java/com/spotify/helios/testing/TemporaryJob.java

```diff
@@ -193,21 +196,12 @@ void deploy() {
         }
       }
 
-      // Wait for jobs to have container ids
```

Comment:
Yea, awaitHasContainerId was a copy and paste of awaitUp, but it didn't need to be. It was awkward having two methods, because you could apply a timeout to both, so you could wait twice as long as the timeout value.